packages/react-cards/package.json

```diff
@@ -47,19 +49,15 @@
   },
   "dependencies": {
     "@microsoft/load-themed-styles": "^1.10.26",
-    "@uifabric/azure-themes": "^7.4.1",
     "@uifabric/file-type-icons": "^7.5.1",
     "@uifabric/foundation": "^7.8.1",
     "@uifabric/set-version": "^7.0.22",
-    "@uifabric/styling": "^7.15.1",
-    "@uifabric/theme-samples": "^7.1.1",
-    "@uifabric/utilities": "^7.29.1",
-    "office-ui-fabric-react": "^7.129.1",
     "tslib": "^1.10.0"
   },
   "peerDependencies": {
     "@types/react": ">=16.8.0 <17.0.0",
     "@types/react-dom": ">=16.8.0 <17.0.0",
+    "office-ui-fabric-react": "^7.129.1",
```

Comment:
You should add OUFR to devDependencies if you take this approach. Also the peerDependencies range is typically wider, and I don't think it will be automatically updated by beachball.